osmomath/go.mod

```diff
@@ -92,8 +92,8 @@ require (
 )
 
 replace (
-	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current branch: v16.x. Direct commit link: https://github.com/osmosis-labs/cosmos-sdk/commit/ed4eb883f2a65bd5343e9d2f6186060f45294cb2
-	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.45.0-rc1.0.20230703010110-ed4eb883f2a6
+	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current branch: osmosis-main. Direct commit link: https://github.com/osmosis-labs/cosmos-sdk/commit/05346fa12992
```

Comment:
OK - I see what the issue is.

We should be using the `v16.x` branch here as `osmosis-main` is only compatible with `main`

However, latest commit on `v16.x` is blocked on this PR: https://github.com/osmosis-labs/osmosis/pull/6341

2 options to resolve:
1. Wait until https://github.com/osmosis-labs/osmosis/pull/6341
2. Use this commit specifically: https://github.com/osmosis-labs/cosmos-sdk/commit/313b4baf0913d384504cbef0dd95704f69c1dd49
- the 2 commits above it have math changes so they would be blocked on #6341

Reply:
I think I tried option #2 and it had the same issue, but we can try again. If that works, it seems like that's the easiest path forward